websites/T/Tiermaker/dist/metadata.json

```
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",

```

Please do tell me where you saw 1.3 being used as the latest version in any tutorial or presence so that that can be fixed, and update to 1.5 while you're at it
```suggestion
    "$schema": "https://schemas.premid.app/metadata/1.5",
```